toolkits/springernature/packages/springer-nature-context/scss/10-settings/colors.scss

```diff
@@ -21,4 +21,15 @@ $colors: (
 	'lilac':  #9d7eb8,
 	'blue': #007e99,
 	'red': #c64040,
-);
\ No newline at end of file
+);
+
+/**
+ * Color and gray scales
+ *
+ */
+
+$page-base: #f8f;
+$greyscale-base: #555;
+$greyscale-min: 20;
+$greyscale-max: 95;
+$greyscale-stops: 5;
```

Comment:
where did these come from?

Reply:
The page-base and grayscale-base colors are from the sketch colour palette for Springer Nature. The min/max/stops are replicated from Springer, and need reviewing with UX. Could remove for now if prefer